### ISSUE TITLE
(Bug report) Search doesn't work

### ISSUE THREAD

**Issue by deajan** [TriliumNext/Notes#287]
### TriliumNext Version

0.90.2-beta

### What operating system are you using?

Windows

### What is your setup?

Local + server sync

### Operating System Version

Windows 11 23H2

### Description

Comparing TriliumNext 0.90.2-beta (latest artifact from https://github.com/TriliumNext/Notes/actions/runs/10149142971/job/28063403365) with Trilium 0.37, I don't get the same search results:

TriliumNext
![image](https://github.com/user-attachments/assets/9303be23-bb0a-414e-9a46-34e0601f71bf)

Trilium
![image](https://github.com/user-attachments/assets/16f720d5-be53-43db-89e8-f8b0c385c0a7)

Note: I don't have any error messages when searching.

**Comment by Nriver** [comment]
Yes, I found this too. I've noticed that some notes become unsearchable. If I slightly modify the note's content, it becomes searchable again. However, if I use Ctrl+Z to revert the change, the note becomes unsearchable once more. The pattern isn't clear, as some notes work without any issues.

**Comment by eliandoran** [comment]
@deajan , this sounds like an ugly issue. 🤔 
If you can isolate the problem, maybe you could send me an anonymized database to check it? 

On my side I can't say I noticed it, however I barely use the quick search feature myself.

**Comment by deajan** [comment]
@eliandoran That will be tough to isolate, especially when the anonymized database will not contain the keyword used in search.

I can reproduce my error on another machine.
The quick search only returns results from node titles. Content doesn't seem to be indexed.
If I happen to slightly modify the content, it will be returned by the quick search.

Data has been synced from server.
Is this sufficient to make a reproducer ?

**Comment by eliandoran** [comment]
@deajan , I believe I've found a fix, please test when you have the time: https://github.com/TriliumNext/Notes/actions/runs/10239059224
